### PR TITLE
create linux introduction

### PR DESCRIPTION
**Pull request by afaaqalam**
I've created first version of Linux introduction.
Looking for feedback! 
@alayek @koustuvsinha  @raisedadead @Rafase282

### CODE REVIEW

**Comment by koustuvsinha** [comment]
looks fine @afaaqalam. just rename the file to 'Linux.md' keeping with our file conventions.

@Rafase282 please check and merge

**Comment by Rafase282** [comment]
@afaaqalam please see the PR I opened with changes to this one and squashing commits.

File: Linux.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Linux
+
+![alt text][logo]
+
+[logo]: https://upload.wikimedia.org/wikipedia/commons/a/af/Tux.png "Linux mascot - Tux"
+
+Linux in simple terms is an operating system. It is very similar to operating systems like Windows and Mac OS.
+[Linus Torvalds](https://en.wikipedia.org/wiki/Linus_Torvalds) built the core of the Linux operating system, known as the kernel. A kernel alone does not make an operating system, but Stallman's [GNU](https://en.wikipedia.org/wiki/GNU) tools were from a project to create an operating system as well--a project that was missing a kernel to make Stallman's operating system complete. Torvalds' matching of GNU tools with the Linux kernel marked the beginning of the Linux operating system as it is known today.
```
Comment (alayek) [inline Linux.md]: > Linux in simple terms is an operating system

This doesn't quite capture the essence of what Linux is. Or what we mean by Linux.

You can write

> Linux is the most popular operating system in the world, which drives almost all the servers, and all your Android phones; among many other devices that you use day to day.

This starts off in a much higher tone.

You can also link to this YouTube [video](https://www.youtube.com/watch?v=yVpbFMhOAwE)

File: Linux.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+### What is kernel?
+
+Kernel is the central part of an operating system that is responsible for interfacing all the applications down to the physical hardware. All operating systems have kernels.
```
Comment (alayek) [inline Linux.md]: This sounds vague. Could you take examples and describe what is a part of kernel and what is not?

For instance, `vim` comes pre-installed with most Linux distro, but not part of the kernel. Also, point to GitHub repo on Kernel.

**Comment by afaaqalam** [comment]
squashed previous commits. @alayek @Rafase282 @koustuvsinha @raisedadead

**Pull request merged**